### PR TITLE
feat(top-app-bar): TopAppBarFixedAdjust variants added

### PR DESCRIPTION
**Pull request by mgr34**
This references #518 and can be closed with the previous merging of PR #520.

I am still unsure about screenshot testing. running `yarn capture` did update the correct part's of `golden.json`  and everything appears as expected from `yarn start`.

---

I signed it

### CODE REVIEW

File: packages/top-app-bar/FixedAdjust.js [hunk 1]
```diff
@@ -29,28 +29,42 @@ const FixedAdjust = (props) => {
...
+    prominent,
+    short,
     ...otherProps
   } = props;
 
+  const base = 'mdc-top-app-bar';
+  const suffix = '-fixed-adjust';
+  const classes = classnames(className, {
+    [base + '--short' + suffix]: short,
+    [base + '--dense' + suffix]: dense && (!prominent),
```
Comment (moog16) [inline packages/top-app-bar/FixedAdjust.js]: nit: can you remove the `()` around prominent and dense

File: packages/top-app-bar/README.md [hunk 2]
```diff
@@ -62,10 +64,21 @@ short | Boolean | Enables short variant.
 shortCollapsed | Boolean | Enables short collapsed variant.
 prominent | Boolean | Enables prominent variant.
 fixed | Boolean | Enables fixed variant.
-dense | Boolean | Enables dense variant.
+dense | Boolean | Enables dense variant. 
+
+> NOTES: As per design guidelines, prominent and dense variants should not be used with short or short collapsed. Additionally, dense variants should only be used on desktop.
```
Comment (moog16) [inline packages/top-app-bar/README.md]: awesome note! thanks :)

File: packages/top-app-bar/README.md [hunk 3]
```diff
@@ -62,10 +64,21 @@ short | Boolean | Enables short variant.
...
 
-> NOTE: As per design guidelines, prominent and dense variants should not be used with short or short collapsed.
+### TopAppBarFixedAdjust
+
+Prop Name | Type | Description
+--- | --- | ---
+className | String | Classes to be applied to the root element.
+dense | Boolean | Enables dense variant. 
+prominent | Boolean | Enables prominent variant.
+short | Boolean | Enables short variant.A
```
Comment (moog16) [inline packages/top-app-bar/README.md]: extra A

File: test/screenshot/golden.json [hunk 4]
```diff
@@ -1,21 +1,21 @@
 {
   "button": "8941ec5719bb5c82418d335f37dd6e4d523bf8a1121b50a4df4c5a784ca41fbe",
-  "card": "b2fd82763c383be438ff6578083bf9009711c7470333d07eb916ab690fc42d31",
+  "card": "4560bf40053939152365ab689aa7892782835d9e4e8c9fff4508debb02afe6d3",
```
Comment (moog16) [inline test/screenshot/golden.json]: only the top app bar ones should have changed. can you revert please.

Comment (mgr34) [reply test/screenshot/golden.json]: eh, that sure is sloppy on my end! I'll probably not get a chance to correct that until Monday.

Comment (moog16) [reply test/screenshot/golden.json]: sounds good! Thanks again

**Comment by moog16** [comment]
tests: https://github.com/material-components/material-components-web-react/pull/538

**Comment by moog16** [comment]
Test passing and CLA signed in: https://github.com/material-components/material-components-web-react/pull/524#issue-238857701 merging!

**Pull request merged**